index.ts

```diff
@@ -1,2 +1,46 @@
-import CashBalances from './data';
+import CashBalances, {CashBalanceReport, ReportSection, Account} from './data';
+
+interface ReportAccountTotals {
+  highest_account_name: string
+  lowest_account: string
```

Comment:
Prefer consistent naming
```suggestion
  highest_account_name: string
  lowest_account_name: string
```